src/styles/_catalog.scss

```diff
@@ -13,10 +14,23 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media (min-width: 488px) and (max-width: 767px) {
```

Comment:
it is better to use or max or min width. In this task there is no need in combination of rules for media